### ISSUE TITLE
make minifig able to pick and leave objects. 

### ISSUE THREAD

**Issue by MAriehamn360-Vitrin2** [cuddlyogre/ExportLDraw#48]
Im trying to make my mini figure to walk and leave a cup but when im following tutorials the object starts to fly away mysteriously. I have a video here that shows the problem. can some one pls help me? 

https://www.youtube.com/watch?v=pE5vtBu0UU8

**Comment by cuddlyogre** [comment]
Try setting Target to the armature then set the bone to the hand bone. You may need to click "Set Inverse"
![image](https://github.com/cuddlyogre/ExportLDraw/assets/3506795/60a6a46f-bbdf-4482-abc2-70b237db5757)

**Comment by MAriehamn360-Vitrin2** [comment]
yes. it worked. how many object is it possible to conect to a figure and how many object can an object to be a child of?

**Comment by cuddlyogre** [comment]
I think an object can have infinite children. 
You should be able to use as many Child Of constraints as you want on an object, but the result will be unpredictable, so it is best to set the influence to 0 when that relationship isn't being used.
